var/spack/repos/builtin/packages/hipcub/package.py

```diff
@@ -80,6 +78,7 @@ class Hipcub(CMakePackage, CudaPackage, ROCmPackage):
     ]:
         depends_on(f"rocprim@{ver}", when=f"+rocm @{ver}")
         depends_on(f"rocm-cmake@{ver}:", type="build", when=f"@{ver}")
+        depends_on(f"hip +cuda@{ver}", when=f"+cuda @{ver}")
```

Comment:
I think the typical order for qualifiers puts version before options, so it would be `hip@{ver} +cuda`, but that's just a matter of style.

Reply:
Then again, it was done the other way for rocprim. Maybe that is the way it's being done now. Nevermind, I suppose.